### PR TITLE
update version of shareplus to the latest version

### PR DESCRIPTION
**Pull request by Faezehyas**
update the version of shareplus to the latest version in the overview.mdx

### CODE REVIEW

File: docs/share_plus/overview.mdx [hunk 1]
```diff
@@ -20,7 +20,7 @@ on iOS.
 dependencies:
   flutter:
     sdk: flutter
-  share_plus: "^{{ plugins.share_plus }}"
```
Comment (miquelbeltran) [inline docs/share_plus/overview.mdx:23]: I am not sure this should be replaced by a hardcoded value

Comment (Faezehyas) [reply docs/share_plus/overview.mdx:23]: Then how do you think it is better to change it?
I use this package in my codes and saw the documentation version is so old.

Comment (miquelbeltran) [reply docs/share_plus/overview.mdx:23]: I believe versions are updated when the website is redeployed, as it is generated automatically from pub.dev.

**Comment by miquelbeltran** [comment]
@mhadaily Do you know how to trigger a re-deploy of the website, so versions are up to date?

**Comment by mhadaily** [comment]
If we can just make a simple change in the website folder it will deploy according to this ´https://github.com/fluttercommunity/plus_plugins/blob/main/.github/workflows/build_deploy_website.yaml´

otherwise, we can run it locally by running ´npm run deploy´ in believe,


I can take care of that, should I deploy now?

**Comment by miquelbeltran** [comment]
Yes, I guess we need to do re-deploys once in a while, if not the versions listed on the site will be outdated

**Comment by mhadaily** [comment]
Ok, let's actually change GitHub action to deploy when we have a new tag available, I'll send a PR today

**Pull request closed without merging**